lxd/instance/drivers/driver_qemu.go

```diff
@@ -4912,6 +4912,8 @@ func (d *qemu) Exec(req api.InstanceExecPost, stdin *os.File, stdout *os.File, s
 		return nil, err
 	}
 
+	d.lifecycle(lifecycle.InstanceExec, log.Ctx{"command": strings.Join(req.Command, " ")})
```

Comment:
Same as container, if the string slice doesn't look horrible, we should keep it.